### PR TITLE
[MNOE-452] Nex!™ Adapter & Assets sync

### PR DESCRIPTION
**Pull request by ouranos**
* Nex!™ adapter for assets sync
* Update Theme Previewer to publish assets

**TODO**
* Nex!™ adapter:
  * Implement restart
  * Implement SSL management
* The ThemeController was not spec'd

### CODE REVIEW

File: core/lib/mno_enterprise/platform_adapters/nex_adapter.rb [hunk 1]
```diff
@@ -1,19 +1,82 @@
 # frozen_string_literal: true
-
-# gem 'nex_client'
-# require 'nex_client'
-# SELF_NEX_API_KEY="some-api-key"
-# SELF_NEX_API_ENDPOINT="https://api-nex-uat.maestrano.io"
+gem 'nex_client', '~> 0.16.0'
```
Comment (alachaum) [inline core/lib/mno_enterprise/platform_adapters/nex_adapter.rb]: Out of curiosity - considering it's a dev dependency are you sure it's going to be included in production?

Comment (ouranos) [inline core/lib/mno_enterprise/platform_adapters/nex_adapter.rb]: No. It's up to the host app to add it in its Gemfile.
I'll add a check when selecting the adapter

Comment (ouranos) [inline core/lib/mno_enterprise/platform_adapters/nex_adapter.rb]: Done

**Comment by ouranos** [comment]
@x4d3 FYI

**Pull request merged**